lib/inline_svg/id_generator.rb

```diff
@@ -1,7 +1,14 @@
 module InlineSvg
   class IdGenerator
-    def self.generate(base, salt)
-      bytes = Digest::SHA1.digest("#{base}-#{salt}")
+    class Randomness
+      require "securerandom"
+      def self.call
+        SecureRandom.hex(10)
+      end
+    end
+
+    def self.generate(base, salt, randomness: Randomness)
```

Comment:
Added this as an injected dependency here so that we can test the interaction between the `generate` method and the whatever provides the random values. I think we can trust the implementation of `SecureRandom` in this case.